### PR TITLE
Core/ts version task 139 login manual test

### PR DESCRIPTION
**Pull request by hypersolo**
## Check list  

- [ ] All Merge Request pipelines should be passed.
- [ ] Make sure all the changes are necessary and correct.
- [ ] Make sure no needed change missing (all necessary changes are made).
- [ ] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [ ] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: src/presentation/http/controllers/auth.controller.ts [hunk 1]
```diff
@@ -20,43 +14,28 @@ export class LoginController extends BaseController {
...
     );
-    const response = {
-      jwt_access_token,
-      user_data,
-    };
     if (uuid) WebSocket.getInstance().addUUID(userAddress, uuid);
-    res.send({ success: true, data: response });
+    res.send({
+      success: true,
+      data: {
```
Comment (WilliamHawkingStephen) [inline src/presentation/http/controllers/auth.controller.ts:28]: - suggestion: It is not necessary for the data to be an object, considering that we have one data response

**Comment by WilliamHawkingStephen** [comment]
- this branch i think should be merge after clean up get user orders and fills 
(because our get user orders and fills don't look right for getting user orders or fills for specific market and network)

**Review by WilliamHawkingStephen — APPROVED** [review]

File: src/presentation/http/controllers/auth.controller.ts (hunk 1 above)
Comment (hypersolo) [reply src/presentation/http/controllers/auth.controller.ts:28]: as we have discussed before all responses have this interface and so this one :))

**Review by franken5tein — APPROVED** [review]

**Review by franken5tein — APPROVED** [review]

File: src/domain/services/auth/auth-handler.ts [hunk 2]
```diff
@@ -42,6 +39,11 @@ export default class AuthHandlerService
     return true;
   }
 
+  addUUID(data: IAuthService.AddUUIDInput): void {
+    const { userAddress, uuid } = data;
+    if (uuid) WebSocket.getInstance().addUUID(userAddress, uuid);
+  }
```
Comment (GreatGaryCooper) [inline src/domain/services/auth/auth-handler.ts:45]: Wrap the if body in curly braces please 😁

**Review by GreatGaryCooper — APPROVED** [review]

**Pull request merged**